### PR TITLE
Update viash

### PR DESCRIPTION
**Pull request by rcannood**
## Issue ticket number

Closes #xxxx 

## Checklist before requesting a review

- [x] I have performed a self-review of my code

- [x] Conforms to the [Contributing guidelines](https://github.com/viash-hub/base/blob/main/CONTRIBUTING.md)

- [x] Proposed changes are described in the [CHANGELOG.md](https://github.com/viash-hub/base/blob/main/CHANGELOG.md)

- [x] I have tested my code with `viash ns test --parallel -q <name or namespace>`

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [ ] New functionality
  - [ ] Major changes
  - [x] Minor changes
  - [ ] Documentation
  - [ ] Bug fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,34 +26,19 @@ We encourage contributions from the community. To contribute:
...
+- **Procedure of adding a component**: \### Step 1: Find a component to
+  contribute
+- **VIASH START**:
+- **VIASH END**: xxx  
+- **VIASH START**:
+- **VIASH END**: arriba  
+- **VIASH START**:
+- **VIASH END**: echo “\> Run xxx with test data”
+- **VIASH START**:
+- **VIASH END**: echo “\> Run arriba with blacklist”
```
Comment (rcannood) [inline README.md:38]: what happened here??

Comment (DriesSchaumont) [reply README.md:38]: Copilot? 😅

Comment (rcannood) [reply README.md:38]: I didn't manually edit the `README.md` manually, though

Comment (rcannood) [reply README.md]: Fixed! I had to change the code to not detect `## VIASH START` as a section name.

**Review by DriesSchaumont — APPROVED** [review]
LGTM!

**Pull request merged**